### ISSUE TITLE
Duplicate enum values generated when only difference is case

### ISSUE THREAD

**Issue by nimish** [mganss/XmlSchemaClassGenerator#204]
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <xs:simpleType name="Enum">
        <xs:restriction base="xs:string">
            <xs:enumeration value="A" />
            <xs:enumeration value="a" />
        </xs:restriction>
    </xs:simpleType>
</xs:schema>
```

generates
```C#
namespace LMAO
{
    using System.Collections;
    using System.Collections.Generic;
    using System.Collections.ObjectModel;
    using System.Xml.Serialization;


    [System.CodeDom.Compiler.GeneratedCodeAttribute("XmlSchemaClassGenerator", "2.0.349.0")]
    [System.SerializableAttribute()]
    [System.Xml.Serialization.XmlTypeAttribute("Enum", Namespace="")]
    public enum Enum
    {

        A,

        [System.Xml.Serialization.XmlEnumAttribute("a")]
        A,
    }
}
```

This appears in real schemas like HL7v3. Presumably the second one should b `a` or `A_1` or something else.

**Comment by mganss** [comment]
This should have been fixed in #190 (2.0.405). Which version are you using?

**Comment by nimish** [comment]
2.0.349.0, oops -- yes it's fixed in 2.0.424.
